packages/suite/src/views/dashboard/AssetsView/AssetTable/AssetRow.tsx

```diff
@@ -221,28 +117,28 @@ export const AssetRowSkeleton = (props: { animate?: boolean }) => {
     const animate = props.animate ?? shouldAnimate;
```

Comment:
Please move `AssetRowSkeleton` into a new file. It does not use styled components anymore, so no need to have it here

Please also improve `(props: { animate?: boolean })` so that it is unified with other components

Reply:
Done